scripts/export_inspector_findings.py

```diff
@@ -169,8 +169,9 @@ def column_alpha(self, col: int) -> str:
     def findings_sort(self, item: tuple[str, list[SummaryType]]) -> tuple[int, str]:
         score = 0
         for summary in item[1]:
-            score += 10 if summary['severity'] == 'CRITICAL' else 0
-            score += 1 if summary['severity'] == 'HIGH' else 0
+            count = len(summary['resources'])
+            score += count * 10 if summary['severity'] == 'CRITICAL' else 0
+            score += count * 1 if summary['severity'] == 'HIGH' else 0
```

Comment:
Don't like how the severity is compared to HIGH even if was determined to be CRITICAL. So

```diff
Index: scripts/export_inspector_findings.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/export_inspector_findings.py b/scripts/export_inspector_findings.py
--- a/scripts/export_inspector_findings.py	(revision a6ab4a3bff53a513c55a599128b3aec2b0a5d10b)
+++ b/scripts/export_inspector_findings.py	(date 1713591303332)
@@ -168,10 +168,10 @@
 
     def findings_sort(self, item: tuple[str, list[SummaryType]]) -> tuple[int, str]:
         score = 0
+        weights = {'HIGH': 1, 'CRITICAL': 10}
         for summary in item[1]:
             count = len(summary['resources'])
-            score += count * 10 if summary['severity'] == 'CRITICAL' else 0
-            score += count * 1 if summary['severity'] == 'HIGH' else 0
+            score += count * weights.get(summary['severity'], 0)
         return score, item[0]
 
     def write_to_csv(self, findings: dict[str, list[SummaryType]]) -> None:
```

… not because it's more efficient (it probably isn't) but because the current technique doesn't scale well to more severities and because it just lacks clarity.